src/monitor-pipeline-group/azext_monitor_pipeline_group/tests/latest/data_files/extendedLocation.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "/subscriptions/b09d4a2d-699b-4c76-8c7a-d97165a77a3b/resourceGroups/kubetest/providers/Microsoft.ExtendedLocation/customLocations/myloc",
```

Comment:
please mock resource subscription id here

Reply:
Replaced with 00000000-0000-0000-0000-000000000000. Looks like the subsciprion Id is already replaced in recording, so changing it here doesn't affect replay.